### PR TITLE
Attempt to invoke virtual method 'Session.getSessionId()' on a null object reference

### PR DESCRIPTION
**Pull request by pedro-cerdera**
Problem occurs a few times when the user logs off shortly after starting the session

### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message

### Solves issue(s)
#137

### CODE REVIEW

**Comment by msach22** [comment]
@pedro-cerdera Thanks for the PR - but it looks like you've bumped version to v0.9.5. Could you please remove the change from the `package.json` because we bump versions after everything has been merged from the release branch to master.

**Comment by pedro-cerdera** [comment]
@msach22 I remove the bump version and insert a new fix to prevent this: 
909bdaab-27f5-4764-8296-4cbf804f12ff

**Comment by msach22** [comment]
@pedro-cerdera Could you please share when you're attempting to send the signal? Does this happen when you connect to the session or is it happening when you attempt to send a signal later on after the session stays connected?

**Comment by pedro-cerdera** [comment]
@msach22 this happen just one time in my tests and happening when I'd connected to the session and log off immediately.

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java [hunk 1]
```diff
@@ -299,7 +299,9 @@ public void removeJSComponentEvents(ReadableArray events) {
     public void sendSignal(ReadableMap signal, Callback callback) {
 
         Session mSession = sharedState.getSession();
-        mSession.sendSignal(signal.getString("type"), signal.getString("data"));
+        if (mSession != null){
+            mSession.sendSignal(signal.getString("type"), signal.getString("data"));
+        }
         callback.invoke();
```
Comment (msach22) [inline android/src/main/java/com/opentokreactnative/OTSessionManager.java]: We should `invoke` an error if we are unable to send a signal and modify the JS code to adapt as well.

**Comment by pedro-cerdera** [comment]
callback added.  We have 154 events of this error in our app.

**Pull request merged**